lib/worker/batcher/bors_toml.ex

```diff
@@ -15,6 +15,9 @@ defmodule BorsNG.Worker.Batcher.BorsToml do
 
   defstruct status: [], block_labels: [], pr_status: [],
     timeout_sec: (60 * 60),
+    # Half an hour. Give up when more time is spent waiting for this amount
+    # between two successive polls.
```

Comment:
This comment is confusing to me. Based on my reading of the PR, maybe this would be better?
```
prerun_timeout_sec controls how long bors will wait for all GitHub status checks to be completed before taking action. If this value is set to 0, bors will not wait for status checks to be completed. If all status checks are not completed within prerun_timeout_sec, bors will return an error message.
```

Reply:
Yeah, I think your suggestion is good.

> If all status checks are not completed within prerun_timeout_sec,

The only thing to note is that I was lazy and the actual timeout is proportional to prerun_timeout_sec but isn't prerun_timeout_sec. prerun_timeout_sec is the amount the last exponential backoff needed to wait.

You can get one timeout from the other by calculating the difference of two geometric series (or the inverse of that).

It was done out of laziness and to have fewer moving parts. But feel free to suggest an improvement to that as well (ideally one that isn't too complicated).

Reply:
Lazy is fine, just so long as it's clear. I think we just need to explain it with words so folks understand their configuration choices.